docs/hub/webhooks-guide-discussion-bot.md

```diff
@@ -0,0 +1,112 @@
+# Webhook guide: build a Discussion bot based on BLOOM
+
+Here's a short guide on how to use HF Webhooks to build a bot that replies to Discussion comments on the Hub with a response generated by BLOOM (using the Inference API).
+
+## Create your webhook in your user profile
+
+First, create your webhook by going to https://huggingface.co/settings/webhooks
+
+- Input a few target repositories that your webhook will run on.
+- You can put a dummy Webhook URL for now, but defining your webhook will let you take a look at the events that are going to be sent to it (and you can replay them, which will be useful for debugging).
+- Input a secret as it will be more secure.
+- subscribe to Community (PR & discussions) events as we are building a Discussion bot
+
+Your webhook will look like this:
+
+![webhook-creation](../assets/webhook-creation.png)
+
+## Create a new "Bot" user profile
+
+In this guide, we create a separate user account to host a Space and to post comments.
+
+![discussion-bot-profile](../assets/discussion-bot-profile.png)
+
+<Tip>
+	When creating a bot that will interact with other users on the Hub, we ask that you clearly label the account as a Bot (see profile screenshot).
+</Tip>
+
+## Create a Space that will react to your webhook
```

Comment:
Could be good to link to the Docker space doc too

Reply:
OK I see there's a link in the Dockerfile 👍

Reply:
but i agree we should add it here too